src/App.js

```diff
@@ -31,7 +31,8 @@ function App() {
       <div className="App-header">
         <div className="App-container">
           <div className="App-section">
-            <h1 className="text-xl">Michael Bloomberg would pay $3.163 billion next year under Elizabeth’s wealth tax.</h1>
+            <title>How Elizabeth Warren Would Make Bloomberg Pay Your Student Loans</title>
```

Comment:
Should this be "Pay Off" or "Student Loan Debt"? Or also it's fine how it is - those just occurred to me

Reply:
Yes, but for headlines we want as short as humanly possible. If we can get away with 'pay' instead of 'pay off' and just 'student loans,' I say let's do that.